pages/index.js

```diff
@@ -0,0 +1,195 @@
+import Head from 'next/head'
+import logger from '../next-pino/logger'
```

Comment:
This logger should be the one attached to the request when doing SSR.
Have we got access to `req` and `res` inside the component?

Reply:
The thing is that once the component is rendered and is served to the browser, we want to be able to send the logs to the server and for that we will need:

`navigator.sendBeacon(${config.serverUrl}/log, blob)`

Reply:
I know. We need a bit of conditionals in here: if there is `req` somewhere, use that. If we are in the browser, use the one with sendBeacon.